public/components/CampaignPerformanceBreakdown/CampaignPerformanceBreakdown.js

```diff
@@ -112,7 +112,15 @@ export default class CampaignPerformanceBreakdown extends React.Component {
       case this.view.PATH:
         return(<CampaignPerformanceBreakdownTable breakdownLabel="Path" analyticsBreakdown={analyticsByPath}/>);
       default:
-        return(<CampaignPerformanceBreakdownTable breakdownLabel="Country" analyticsBreakdown={analyticsByCountryCode}/>);
+        return(null);
+    }
+  }
+
+  renderLocationNavItemIfGlobal() {
```

Comment:
We don't want to show the country breakdown on campaigns where the territory is not global.